hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/rollback/RollbackUtils.java

```diff
@@ -127,7 +129,7 @@ static HoodieRollbackStat mergeRollbackStat(HoodieRollbackStat stat1, HoodieRoll
   public static List<ListingBasedRollbackRequest> generateRollbackRequestsUsingFileListingMOR(HoodieInstant instantToRollback, HoodieTable table, HoodieEngineContext context) throws IOException {
     String commit = instantToRollback.getTimestamp();
     HoodieWriteConfig config = table.getConfig();
-    List<String> partitions = FSUtils.getAllPartitionPaths(context, config.getMetadataConfig(), table.getMetaClient().getBasePath());
+    List<String> partitions = FSUtils.getAllPartitionPaths(context, HoodieMetadataConfig.newBuilder().enable(false).build(), table.getMetaClient().getBasePath());
```

Comment:
This comment is not specific to the patch. Something to consider:
Since there are many places where some metadata configs are hardcoded, would it be better to extract them to constants? For e.g. in this case we can introduce ROLLBACK_METADATA_ENABLE_DEFAULT = false and reuse the constant wherever needed. Add a small comment wherever we declare the constant why the default value makes sense. This would help to keep code maintainable and readable.

Reply:
I found another method in FSUtils which takes an arg for metadata. For now re-used that. but will watch for future usages. good suggestion!